### ISSUE TITLE
Version update release for dropwizard 2.0.2

### ISSUE THREAD

**Issue by chris-orchard** [xvik/dropwizard-guicey-ext#16]
Hi there's a regression in dropwizard 2.0.0 that's fixed in 2.0.2.

Can you make a release that uses the latest versions please?

`dropwizard-guicey 5.0.0-0` depends on `dropwizard-jdbi3:2.0.0`, which in turn uses `jdbi3-core:3.11.1`.

Both of these versions get marked as conflicts in my build as the newer versions are included by 2.0.2 versions of other dropwizard modules.

As shown in my gralde build scan:
![image](https://user-images.githubusercontent.com/12509945/75032345-46599d00-54a0-11ea-9ea0-c4ef09e23d56.png)

**Comment by xvik** [comment]
Hi!

The recommended way of overriding dependencies is through BOMs:

```groovy
plugins {
    ...
    id 'io.spring.dependency-management' version '1.0.9.RELEASE'
}

dependencyManagement {
        imports {
            mavenBom 'ru.vyarus.guicey:guicey-bom:5.0.0-0'
            mavenBom 'io.dropwizard:dropwizard-bom:2.0.2'
            mavenBom 'io.dropwizard:dropwizard-dependencies:2.0.2'
        }
}

dependencies {
    implementation 'ru.vyarus.guicey:guicey-jdbi3'
}
```

Probably, you already do it like this. And, even in this case, gradle will complain about versions.
But still, this way, all versions will be correct (dependency version from the latest applied BOM always wins).

So, overall, it's not a critical problem and there is a way to use correct versions. I think the behavior will be the same with gradle native BOMs support (platforms).

The reason why I don't want to do releases for each new dropwizard version is simply time: guicey has 3 repositories now (and there will be more). On each release, I need to manually update and release them. Plus, I need to update and publish updated documentation. Overall, about 1h (more in case of publication issues) to perform a release.

Sorry. For now, I will not do releases only to update dependencies without real need. This may change in the future (as I also always have to override dependencies manually so I feel that need too).

I plan to do guicey release with junit5 support within a month and it will, of course, depend on the latest dropwizard.

**Comment by xvik** [comment]
Released [5.0.1](https://github.com/xvik/dropwizard-guicey/releases/tag/5.0.1), ext [5.0.1-1](https://github.com/xvik/dropwizard-guicey-ext/releases/tag/5.0.1-1)

Updated [docs](http://xvik.github.io/dropwizard-guicey/5.0.1/)
